developer-docs-site/static/examples/python/first_nft.py

```diff
@@ -42,18 +46,25 @@ def create_token(
             supply: int,
             uri: str,
     ):
+        mutate_setting = [False, False, False, False, False]
         payload = {
             "type": "script_function_payload",
-            "function": f"0x1::token::create_unlimited_token_script",
+            "function": f"0x1::token::create_token_script",
             "type_arguments": [],
             "arguments": [
                 collection_name.encode("utf-8").hex(),
                 name.encode("utf-8").hex(),
                 description.encode("utf-8").hex(),
-                True,
                 str(supply),
+                str(U64_MAX),
```

Comment:
Can we please reconsider option

Reply:
will make the SDK this parameter optional. However, this tutorial probably would like to keep this way until it is restructured using the SDK